package.json

```
@@ -25,13 +25,13 @@
   ],
   "peerDependencies": {
     "@google-cloud/pubsub": "^2.0.0",
```

```suggestion
    "@google-cloud/pubsub": "^2.0.0 || ^3.0.0",
```

Can you add support for 3.0.0?